test/integration-metrics.test.js

```diff
@@ -197,6 +197,25 @@ describe('Metrics SDK Integration Tests', function () {
     expect(response.content.mimeType).toMatch(/application\/json(;\s?charset=utf-8)?/);
   });
 
+  it('should include an _id UUID in har payload', async function () {
+    await fetch(`http://localhost:${PORT}`, { method: 'get' });
+
+    const [, body] = await getRequest();
+    const [har] = body;
+    expect(typeof har._id).toBe('string');
+  });
+
+  it.todo('should add `x-documentation-url` to response headers', async function () {
```

Comment:
Should we bring this test in? Are there any other SDKs that don't yet support this?

```suggestion
  it('should add `x-documentation-url` to response headers', async function () {
```

Reply:
Ruby doesn't currently support it. There's a ticket tracking it in the `metrics-sdk` project! Otherwise, we'll need to integrate some kind of mocking into the integration tests to support this across the board.